extensions/wikia/CreateNewPageFlow/scripts/flowTracking.js

```diff
@@ -0,0 +1,34 @@
+define('flowTracking', ['wikia.cookies', 'wikia.tracker', 'mw'], function (cookies, tracker, mw) {
+	var cookieName = 'CNPFlowLabel',
+		track = tracker.buildTrackingFunction({
+			action: tracker.ACTIONS.CLICK,
+			trackingMethod: 'analytics',
+			category: 'create-new-page-flows'
+		});
+
+	function beginFlow(label) {
+		cookies.set(cookieName, label, { path: mw.config.get('wgCookiePath') });
+		track({
+			label: 'begin-' + label
+		});
+	}
+
+	function endFlow() {
```

Comment:
Per https://github.com/Wikia/app/pull/11504/files#diff-b8f12ba9e03cc0196a8a5ee25614c461R9 - maybe it'd be good to pass flow name as param here too?